linkerd/app/outbound/src/tcp/opaque_transport.rs

```diff
@@ -20,16 +18,27 @@ pub struct OpaqueTransport<S> {
 }
 
 impl<S> OpaqueTransport<S> {
-    pub fn layer() -> impl layer::Layer<S, Service = Self> + Copy {
-        layer::mk(|inner| OpaqueTransport { inner })
+    pub fn layer() -> impl svc::Layer<S, Service = Self> + Copy {
+        svc::layer::mk(|inner| OpaqueTransport { inner })
+    }
+
+    /// Determines whether the connection has negotiated support for the
+    /// transport header.
+    #[inline]
+    fn negotiated_header<I: tls::HasNegotiatedProtocol>(io: &I) -> bool {
```

Comment:
nit/TIOLI: since this returns a bool i might prefer `has_negotiated_header` or something?